packages/gestalt/src/sharedSubcomponents/FormHelperText.tsx

```diff
@@ -14,12 +16,35 @@ type Props = {
   size?: SizeType;
 };
 
-const applyDensityMargin = (size?: SizeType): 1 | 2 => (size === 'sm' || size === 'md' ? 1 : 2);
-
 export default function FormHelperText({ id, currentLength, text, maxLength, size }: Props) {
+  const isInVRExperiment = useInExperiment({
+    webExperimentName: 'web_gestalt_visualRefresh',
+    mwebExperimentName: 'web_gestalt_visualRefresh',
+  });
+
   return (
     // id is required for all helper texts accompanying an individual form element, not for groups of form elements such as RadioGroup.
-    <Box id={id} marginTop={applyDensityMargin(size)}>
+
+    <div
+      className={classnames({
+        // sm
+        [styles.sm_startPadding]: !isInVRExperiment && size === 'sm',
+        [styles.sm_topPadding]: !isInVRExperiment && size === 'sm',
+        [styles.vr_sm_startPadding]: isInVRExperiment && size === 'sm',
+        [styles.vr_sm_topPadding]: isInVRExperiment && size === 'sm',
+        // md
+        [styles.md_startPadding]: !isInVRExperiment && size === 'md',
+        [styles.md_topPadding]: !isInVRExperiment && size === 'md',
+        [styles.vr_md_startPadding]: isInVRExperiment && size === 'md',
+        [styles.vr_md_topPadding]: isInVRExperiment && size === 'md',
+        // lg
+        [styles.lg_startPadding]: !isInVRExperiment && size === 'lg',
+        [styles.lg_topPadding]: !isInVRExperiment && size === 'lg',
+        [styles.vr_lg_startPadding]: isInVRExperiment && size === 'lg',
+        [styles.vr_lg_topPadding]: isInVRExperiment && size === 'lg',
+      })}
+      id={id}
+    >
```

Comment:
![image](https://github.com/user-attachments/assets/c86b20e3-2fb4-4d11-85e9-6cf92c57a19f)
![image](https://github.com/user-attachments/assets/7430c249-846d-4766-b135-3b0c86ab2cec)

The color on disabled states is 'disabled'